src/app/components/filebrowsermvs/filebrowsermvs.component.ts

```diff
@@ -168,7 +178,11 @@ export class FileBrowserMVSComponent implements OnInit, OnDestroy {//IFileBrowse
       }}
     ];
   }
-
+  openInNewTab(rightClickedFile: any) {
+    const baseURI = `${window.location.origin}${window.location.pathname}`;
+    const newWindow = window.open(`${baseURI}?pluginId=org.zowe.editor:data:{"type":"openDataset","name":"${encodeURIComponent(rightClickedFile.data.path)}","toggleTree":true}`, '_blank');
```

Comment:
I see a problem with this but we can solve it in two ways:

1. This is Editor-reliant code. We can have code used for the Editor, like `(nodeClick)="onNodeClick($event)"` but the FT is a widget that isn't Editor-reliant, it's actually implemented in other apps. What this should be is an event, maybe
`newTabClick="onNewTabClick($event)`  A listener from the Editor will listen, and choose what to do by interpreting it (by opening an Editor tab). A different app, like the File Transfer App, may choose to do something else. Maybe the FTA does decide to open it in the Editor too, but it's for plugins to decide, not the File Tree.

2. You can make an Editor specific right-click option. It won't be reliant on the Editor, because we can use the Zowe PluginManager to check if the Editor plugin is installed. If it's not installed, we simply don't spawn this option in the right click menu. If the Editor does exist, we can call it maybe,
"Open in Standalone Editor Tab" (not "Open in New Editor Tab" because this implies we are referring to a tab within the Editor app)